### ISSUE TITLE
Husky not skipping install on 0.15.0-rc.3

### ISSUE THREAD

**Issue by csi-lk** [typicode/husky#231]
As per the documentation:

> If you don't want husky to automatically install Git hooks, simply set HUSKY_SKIP_INSTALL environment variable to true.

Have tried doing this in both the env and on the CLI

```bash
> husky@0.15.0-rc.3 postinstall /code/node_modules/husky
> node lib/installer/bin install

/code/node_modules/husky/node_modules/execa/index.js:347
        throw err;
        ^

Error: Command failed: git rev-parse --git-dir
fatal: Not a git repository (or any of the parent directories): .git


    at makeError (/code/node_modules/husky/node_modules/execa/index.js:169:9)
    at Function.module.exports.sync (/code/node_modules/husky/node_modules/execa/index.js:338:15)
    at Object.<anonymous> (/code/node_modules/husky/lib/installer/bin.js:11:16)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
    at Function.Module.runMain (module.js:684:10)
    at startup (bootstrap_node.js:187:16)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! husky@0.15.0-rc.3 postinstall: `node lib/installer/bin install`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the husky@0.15.0-rc.3 postinstall script.
```

```bash
ERROR: Service 'node' failed to build: 
The command '/bin/sh -c cd /code && HUSKY_SKIP_INSTALL=true npm install' 
returned a non-zero code: 1
```

This is occurring as part of our build, so is within a docker container `node:8`

### To Reproduce

Dockerfile:

```dockerfile
FROM node:8
ENV HUSKY_SKIP_INSTALL=true
COPY package*.json /code/
RUN cd /code && \
    HUSKY_SKIP_INSTALL=true npm install
```

**Comment by typicode** [comment]
Hi @csi-lk,

Thank you for the issue, I've published `v0.15.0-rc.4`, available under `@next` tag. It should fix the issue.
Instead of returning with a non-zero code, it should just display an error message but exit with 0.

**Comment by janeklb** [comment]
Still wasn't able to do this with `0.15.0-rc.13`, but `husky@1.0.0-rc.7` does the trick
